apps/betterangels-e2e/.detoxrc.json

```diff
@@ -41,6 +46,12 @@
       "type": "android.apk",
       "build": "npx nx run betterangels:build --platform android --profile preview --wait --local --no-interactive --output=../../apps/betterangels/dist/BetterAngels.apk",
       "binaryPath": "../../apps/betterangels/dist/BetterAngels.apk"
+    },
+    "android.test": {
+      "type": "android.apk",
+      "build": "npx nx run betterangels:build --platform android --profile test --wait --local --no-interactive --output=../../apps/betterangels/dist/apk-build.tar.gz && mkdir -p ../../apps/betterangels/dist/apk-build/ && tar -xzf ../../apps/betterangels/dist/apk-build.tar.gz -C ../../apps/betterangels/dist/apk-build/",
+      "binaryPath": "../../apps/betterangels/dist/apk-build/release/app-release.apk",
+      "testBinaryPath": "../../apps/betterangels/dist/apk-build/androidTest/debug/app-debug-androidTest.apk"
```

Comment:
I put both the android and ios builds in `/dist`, though we probably may need a better location that is test specific. think using `dist` may conflict with other things, but first let's see if detox is the way

Reply:
@vecchp - any suggestions for alternate placements of these builds?
* `apps/betterangels/test`
* `apps/betterangels/detox`
* `apps/betterangels/.build`
* `apps/betterangels/.build/detox`